src/components/RepoBar/style.css

```diff
@@ -0,0 +1,55 @@
+.repo-bar {
+  width: 700px;
+  height: 100px;
+  padding: 20px;
+  color: rgb(31, 31, 31);
+  background-color: rgba(255, 166, 0, 0.856);
+  border-radius: 20px;
+  display: flex;
+  border: 3px solid rgb(240, 240, 240);
+  margin: 20px;
+}
+
+.repo-bar:hover {
```

Comment:
love that it changes colour when you hover

Reply:
❤️